assets/samplesheet.csv

```diff
@@ -1,4 +1,4 @@
-sample,fastq_1,fastq_2,reference_assembly
-SAMPLE1,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/a_1.fastq,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/a_2.fastq,https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/reference.fasta
-SAMPLE2,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/b_1.fastq,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/b_2.fastq,
-SAMPLE3,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/c_1.fastq,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/c_2.fastq,
+sample,fastq_1,fastq_2,reference_assembly,metadata_1,metadata_2,metadata_3,metadata_4,metadata_5,metadata_6,metadata_7,metadata_8
+SAMPLE1,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/a_1.fastq,https://raw.githubusercontent.com/phac-nml/snvphyl-galaxy-cli/development/example-data/fastqs/a_2.fastq,https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/reference.fasta,,,,,,,,
```

Comment:
Done here: [c5a3457](https://github.com/phac-nml/snvphylnfc/pull/22/commits/c5a345751ab67690a514970ddb0c0e09199767de)
Also updated the test.config input path to "${projectDir}/assets/samplesheet.csv" for the time being. This can be reverted once merged into `dev`.